src/App.jsx

```diff
@@ -2,18 +2,20 @@ import React from "react";
 import DisplayArticlesList from "./components/DisplayArticlesList";
 import { Switch, Route } from "react-router-dom";
 import DisplayArticle from "./components/DisplayArticle";
-import Header from "./components/Header"
-import Footer from './components/Footer'
+import Header from "./components/Header";
+import Footer from "./components/Footer";
+import CategoryMenu from "./components/CategoryMenu";
 
 const App = () => {
   return (
     <>
-    <Header/>
+      <Header />
+      <CategoryMenu />
       <Switch>
         <Route exact path="/" component={DisplayArticlesList}></Route>
         <Route exact path="/articles/:id" component={DisplayArticle}></Route>
```

Comment:
Why did you remove the category Route?

Reply:
It didn't work, so we're still trying to figure it out. Yet, it doesn't need to be in this user story so we deleted it for now.